src/omv/sensors/paj6100.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * This file is part of the OpenMV project.
+ * Copyright (c) 2019 Lake Fu <lake_fu@pixart.com>
+ * This work is licensed under the MIT license, see the file LICENSE for details.
+ *
+ * PAJ6100 driver.
+ *
+ */
+#ifndef __PAJ6100_H__
+#define __PAJ6100_H__
+#include <stdbool.h>
+#include "sensor.h"
+#define PAJ6100_XCLK_FREQ 6000000
+
+bool findPaj6100(sensor_t *sensor);
```

Comment:
```suggestion
bool paj6100_detect(sensor_t *sensor);
```

Reply:
If we want to change this function name, sensor.c should update too.
But it seems I cannot edit PR, can you help about that?

Reply:
Just force push an update. Or do another commit.